### PR TITLE
[Doc issue][Update guest member permissions]

### PR DESCRIPTION
**Pull request by v-abirade**

### CODE REVIEW

File: api-reference/beta/api/team-post-members.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Namespace: microsoft.graph
 Add a new [conversationMember](../resources/conversationmember.md) to a [team](../resources/team.md).
 
 ## Permissions
+
+> [!NOTE]
+> You can't add the guest members to a team with app permissions.
```
[the remark this answers is not in the document]
Comment (v-abirade) [reply api-reference/beta/api/team-post-members.md:20]: Hi @Lauragra,

The suggestion is in passive voice: "Using application permissions to add guest members to a team is not supported.", I have updated the same information in active voice: You can't add guest members to a team by using application permission.
Request you to please merge the PR.

Regards,
Abira

[the remark this answers is not in the document]
Comment (Lauragra) [reply api-reference/beta/api/team-post-members.md]: In this example, I prefer passive voice over telling the user they can't do something - the scenario is actually not supported. In some cases, in developer content passive voice is warranted; see the following guidance:
https://docs.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice 

Also, my first comment was about the additional detail around identifying team members who are guests. Do you know the answer to that? We might need additional context or a link in the note text.

[the remark this answers is not in the document]
Comment (v-abirade) [reply api-reference/beta/api/team-post-members.md]: I have provided link to topic on guest members. Sentence updated as suggested.

File: api-reference/v1.0/api/team-post-members.md [hunk 2]
```diff
@@ -21,7 +22,10 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application| TeamMember.ReadWrite.All |
 
-> **Note**: Permissions marked with * use [resource-specific consent](/microsoftteams/platform/graph-api/rsc/resource-specific-consent).
+> [!NOTE]
+> * Using application permissions to [add guest members](/microsoft-365/admin/add-users/about-guest-users?view=o365-worldwide&preserve-view=true) to a team is not supported.
+> * Permissions marked with * use [resource-specific consent](/microsoftteams/platform/graph-api/rsc/resource-specific-consent).
```
Comment (Lauragra) [inline api-reference/v1.0/api/team-post-members.md:27]: I don't see any permissions with an * in the table; remove this bullet?

Comment (v-abirade) [reply api-reference/v1.0/api/team-post-members.md]: Removed the bullet

**Review by Lauragra — APPROVED** [review]

**Pull request merged**